### PR TITLE
remove redundant offensive Module

### PR DESCRIPTION
**Pull request by DysonBreakstone**
This removes the redundant Offensive Module

### CODE REVIEW

File: lib/offensive.rb [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
-    @games.map do |game|
-      offense[game.away_team_id] << game.away_goals
-      offense[game.home_team_id] << game.home_goals
-    end
-    average = {}
-    offense.each do |id, goals|
-      average[id] = goals.sum / goals.count.to_f
-    end
-    average
+  def offensive(*away_or_home)
```
Comment (margdaly) [inline lib/offensive.rb:3]: this is such a small detail and I don't think it matters but is the asterisk in '*away_or_home' on purpose?

File: lib/offensive.rb [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+    offense = Hash.new
+
+    applicable_games = @game_teams.select { |game_team|
+      away_or_home.include?(game_team.hoa)
+    }
+
+    @teams.each { |team|
+      this_teams_goals = applicable_games.select{ |game_team|
```
Comment (margdaly) [inline lib/offensive.rb:11]: I really like how you labeled each variable- it helped me a lot to follow along

File: lib/offensive.rb [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
+        game_team.team_id == team.team_id
+        }.sum{|game_team| 
+          game_team.goals.to_f}
+      this_teams_games = applicable_games.select{|game_team| 
+        game_team.team_id == team.team_id}.length
+      offense[team.team_id] = (this_teams_goals / this_teams_games.to_f)
```
Comment (margdaly) [inline lib/offensive.rb:17]: I'm so impressed with the refactor! 
Combining the offensive modules was really great

**Review by margdaly — APPROVED** [review]
Great working drying out the code! 
- merging the offensive modules was beautifully done and makes the code much more 'dry'
- I really appreciated how you labeled the variables in the module! very clear to follow

**Review by AlejandroLopez1992 — APPROVED** [review]

**Pull request merged**